snuba/querylog/query_metadata.py

```diff
@@ -201,7 +201,7 @@ def to_dict(self) -> snuba_queries_v1.QueryMetadata:
             "sql_anonymized": self.sql_anonymized,
             "start_timestamp": start,
             "end_timestamp": end,
-            "stats": self.stats,
+            "stats": self.stats,  # type: ignore
```

Comment:
@untitaker I'm not sure how to fix this. I couldn't get TypedDict doesn't work nicely with omitted keys (and we obviously don't want to define all of the keys in the schema as we don't care about some of them).

Reply:
swapped to cast